README.md

```diff
@@ -1,4 +1,30 @@
-triples
+Triples
 =======
 
+The challenge is to find 10 sets of 3 matching cards.
+
+Great workout for the memory!
+
+How to use?
+-----------
+
+Triples is part of the Sugar desktop. Please refer to;
+
+* [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/),
+* [How to use Sugar](https://help.sugarlabs.org/),
+* [Download Triples](https://activities.sugarlabs.org/en-US/sugar/addon/4461).
```

Comment:
As activities.sugarlabs.org does not and likely will not contain Python 3 activities, I'd expect this line to be removed during Port to Python 3 rather than added.